laske_export/document/invoice_sales_order_adapter.py

```diff
@@ -262,13 +262,9 @@ def set_line_item_common_values(
             # In case of collateral ("Rahavakuus") receivable type, populate the
```

Comment:
Should this comment be removed or kept? Either way, the function logic is ok.

Reply:
Nicely noticed, I will fix the comment :+1: